code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -67,12 +67,10 @@
 		PF.flags_can_pass_all = PASS_ALL^PASS_OVER_THROW_ITEM
 
 /mob/living/carbon/xenomorph/facehugger/Life(delta_time)
-	if(!client && !aghosted && away_timer > XENO_FACEHUGGER_LEAVE_TIMER)
+	if(stat != DEAD && !lying && !(mutation_type == FACEHUGGER_WATCHER) && !(locate(/obj/effect/alien/weeds) in get_turf(src)))
 		// Become a npc once again
 		new /obj/item/clothing/mask/facehugger(loc, hivenumber)
 		qdel(src)
-		return
-	if(stat != DEAD && !lying && !(locate(/obj/effect/alien/weeds) in get_turf(src)))
 		adjustBruteLoss(1)
```

Comment:
why are we adjusting brute loss after qdel(src)

Reply:
alright now I 100% fixed the conflict